core/block.js

```diff
@@ -1724,6 +1725,42 @@ Blockly.Block.prototype.setCommentText = function(text) {
   }
 };
 
+/**
+ * Set this Block's breakpoint as enabled or disabled.
+ * @param {?boolean} enable A boolean definining if the breakpoint should be enabled or disabled.
```

Comment:
Should be `{boolean}` I think. 

The question mark [means nullable](https://github.com/google/closure-compiler/wiki/Annotating-JavaScript-for-the-Closure-Compiler#nullable-type)